src/js/edu-benefits/components/personal-information/DependentInformationFields.jsx

```diff
@@ -21,7 +16,7 @@ export default class DependentInformationFields extends React.Component {
             value={this.props.data.serviceBefore1977.married}
             onValueChange={(update) => {this.props.onStateChange('serviceBefore1977.married', update);}}/>
         <ErrorableRadioButtons
-            label="Do you have any children who are under age 18, or over age 18 but under age 23, not married and attending school, or of any age permanently helpless for mental or physical reasons?"
+            label="Do you have any children who are under age 18, or over age 18 but under 23, not married and attending school, or of any age permanently disabled for mental or physical reasons?"
```

Comment:
I've been adding content questions as per-chapter issues in edu-benefits, with a checklist. I think that's been a pretty good method for tracking. I can add it.